project1/src/components/Article/ArticleMeta.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
```

Comment:
what is the difference between `ArticleInfo` and `ArticleMeta`? the names are not descriptive!